ntbs-service/Models/Enums/Result.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace ntbs_service.Models.Enums
+{
+    public enum Result
+    {
+        Positive,
+        Negative,
+        Awaiting
+    }
+
+    public static class ResultEnumHelper
+    {
+        public static List<Result> GetAll()
```

Comment:
I feel like we have a similar method elsewhere - is it possible to rewrite as a generic method taking type T : enum?

Reply:
Yes indeed we have one for status below!

Reply:
good catch! :)